tools/report-converter/tests/unit/kerneldoc_output_test_files/files/sample.c

```diff
@@ -0,0 +1,9 @@
+// This is a sample file
```

Comment:
I think it would be good to highlight here in this comment that the Kernel-Doc will give a warning to this comment.
```suggestion
// Expected warning: Kenel-Doc.
```